DEPRECATIONS.md

```diff
@@ -0,0 +1,51 @@
+# ember-m3
+
+## Deprecations
+
+### 4.x
+
+#### Model Native Properties
+
+**ID**: 'm3.model.native-property',
+
+**Until**: 5.0
+
+M3 is adding support for not having to use `.get()` and `.set()` on models. If you do not currently
+do `.` access on your m3 models, i.e. you don't have code that does `someModel.someInternalState = value`
+it is safe to turn on the native property access support by definining `useNativeProperties` method
+in your schema:
+
+```js
+  useNativeProperties(/* modelName */) {
+    return true;
+  }
+
+```
+This will allow you to use `.` instead of `.get()` and `.set()` with your m3 models.
+
+If you do currently access your models using `.`, you will need to migrate away from doing that, as
+once you enable native property access, those values will be treated as attributes to be sent to 
+the server.
+
+For a smooth and piecemeal migration, you can define
+
+```js
+  useNativeProperties(/* modelName */) {
+    return false;
+  }
+```
+in your schema. 
```

Comment:
For a smooth and piecemeal migration, you can define the following in your schema:

```js
  useNativeProperties(/* modelName */) {
    return false;
  }
```

Reply:
```suggestion
For a smooth and piecemeal migration, you can define the following in your schema:
```js
  useNativeProperties(/* modelName */) {
    return false;
  }
```